Documentation/Contributing/development-environment.md

```diff
@@ -58,3 +58,37 @@ Use [helm.sh](https://github.com/rook/rook/blob/master/tests/scripts/helm.sh) to
 
 !!! note
     If Helm is not available in your `PATH`, Helm will be downloaded to a temporary directory (`/tmp/rook-tests-scripts-helm`) and used from that directory.
+
+
+## Using local Rook image on minikube cluster
+
+Developers can test quickly their changes by building and using the local Rook image
+on their minikube cluster as following:
+
+1) Set minikube Docker environment variables:
+```console
+eval $(minikube docker-env -p minikube)
+```
+
+2) Build your local Rook image:
+```console
+cd <your_rook_src_direcotry>
+make
+```
+
+3) This will generate a Rook image which label has the format `build-xxxxxxxx/ceph-amd64`. Following
+is an example output from the build process:
+```console
```

Comment:
Suggestion to remove the output logs. The logs might change in future.

Reply:
@sp98 I put them because as of my personal experience it's hard to know where this tag is coming from. Giving a sample from the output makes it easy for the user to find the tag. If it changes in the future we should update the docs :)

Reply:
Instead of showing the build output, how about showing the image name as a result of `docker images`?

Reply:
using the argument BUILD_REGISTRY (suggested by Blaine) to specify the name of the build the image name is deterministic and there's no need to put additional context so I'm just removing the build output.